db/migrate/20200716130243_create_submissions.rb

```diff
@@ -0,0 +1,11 @@
+class CreateSubmissions < ActiveRecord::Migration[6.0]
+  def change
+    create_table :submissions, id: :uuid do |t|
+      t.string :state, default: :inactive
```

Comment:
Wondering whether this would be more useful as an `enum`?

If we need to manually create a submission in the console, a typo in the state would cause the `I18n.t` call to blow the page up potentially.